docs/send-data/kubernetes/install-helm-chart.md

```diff
@@ -31,7 +31,7 @@ If you don’t already have a Sumo account, you can create one by clicking the F
 The following are required to set up Sumo Logic's Kubernetes collection.
 
 - An [Access ID and Access Key](/docs/manage/security/access-keys/) with
-  [Manage Collectors](/docs/manage/users-roles/roles/role-capabilities#data-management) capability.
+  [Manage Content and Manage Collectors](/docs/manage/users-roles/roles/role-capabilities#data-management) capabilities.
```

Comment:
```suggestion
  [Manage Content and Manage Collectors capabilities](/docs/manage/users-roles/roles/role-capabilities#data-management).
```

Reply:
@kimsauce / @jpipkin1, please pick a version that you prefer. Thanks @sumo-drosiek for suggestion 🙇

Reply:
@dkarabin-sumo, it should say "Manage Content and Manage Collectors capabilities".